cartographer/mapping_2d/pose_graph.h

```diff
@@ -102,8 +102,10 @@ class PoseGraph : public mapping::PoseGraph {
   std::vector<std::vector<int>> GetConnectedTrajectories() override;
   mapping::PoseGraph::SubmapData GetSubmapData(
       const mapping::SubmapId& submap_id) EXCLUDES(mutex_) override;
-  mapping::MapById<mapping::SubmapId, mapping::PoseGraph::SubmapData>
+  mapping::MapById<mapping::SubmapId, mapping::PoseGraphInterface::SubmapData>
   GetAllSubmapData() EXCLUDES(mutex_) override;
+  mapping::MapById<mapping::SubmapId, SubmapPose> GetAllSubmapPoses()
```

Comment:
consistency: mapping::PoseGraphInterface::SubmapPose, also in _3d/pose_graph.h

Reply:
As discussed SubmapData is already defined in the implementing class that's we need to add the explicit namespace to disambiguate. Here this is not necessary. And we usually don't specify it when not necessary.